bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolderRenderer.java

```diff
@@ -1696,9 +1696,8 @@ Color getFillColor() {
 
 	private Font getChevronFont(Display display) {
 		if (chevronFont == null) {
-			Point dpi = display.getDPI();
-			int fontHeight = 72 * CHEVRON_FONT_HEIGHT / dpi.y;
```

Comment:
The static variable `CHEVRON_FONT_HEIGHT` is now unused and thus needs to be removed.